src/reason_parser.mly

```diff
@@ -4095,12 +4228,16 @@ val_ident:
   | PERCENT                                     { "%" }
   | LESSGREATER                                 { "<>" }
   | LESSDOTDOTGREATER                           { "<..>" }
+  | SLASHGREATERLESS                            { "/><" }
```

Comment:
Will explain in an upcoming commit, reason is that we need it for removing spaces between tags.